nexus/src/app/sagas/instance_create.rs

```diff
@@ -329,10 +329,25 @@ async fn sic_alloc_server(
     sagactx: NexusActionContext,
 ) -> Result<Uuid, ActionError> {
     let osagactx = sagactx.user_data();
-    let params = sagactx.saga_params::<Params>()?;
+
+    // ALLOCATION POLICY
+    //
+    // NOTE: This policy can - and should! - be changed.
+    //
+    // See https://rfd.shared.oxide.computer/rfd/0205 for a more complete
+    // discussion.
+    //
+    // Right now, allocate an instance to any random sled agent.
     osagactx
-        .alloc_server(&params.create_params)
+        .nexus()
+        .random_sled_id()
```

Comment:
There are a lot of factors we should be taking into account eventually, but are not in this PR.

I think that's okay - this PR is an improvement over `main` - but I wanted to get this in writing:

- There's no consideration for "health of the sled" here, other than "time_deleted = Null". If the sled is rebooting, in a known unhealthy state, etc, we'd currently provision it here. I don't think this is a trivial fix, but it's work we'll need to account for eventually.
- This is selecting a random sled from *all sleds in the cluster*. For multi-rack, this is going to fling the sled to an arbitrary system. Maybe that's okay, but worth knowing about explicitly.

Reply:
I put those factors in a comment (plus one about anti-affinity) in b11849190aace4db50370a7224f11f3af16cb45e